src/config.rs

```diff
@@ -1,5 +1,5 @@
 pub const REMOVE_TOMBSTONE: &[u8] = b"%_%_%_%<!(R|E|M|O|V|E|D)!>%_%_%_%_";
-pub static DATA_FILE_GLOB_FORMAT: &'static str = "data.*";
```

Comment:
the `'static` here is redundant, its a static binding so of course its a static lifetime on the string slice.